packages/target-tauri/src-tauri/build.rs

```diff
@@ -17,7 +17,43 @@ fn main() {
     println!("cargo:rustc-env=BUILD_TIME_STAMP={build_time_stamp}");
     println!("cargo:rustc-env=BUILD_INFO_GIT={}", get_git_ref());
 
-    tauri_build::build()
+    tauri_build::try_build(tauri_build::Attributes::default().app_manifest(
+        tauri_build::AppManifest::new().commands(&[
```

Comment:
a comment about why and how this works would be nice here
```suggestion
        // This adds capabilities for the commands so we can specify which window has access to what in the tauri configuration.
        // see also https://docs.rs/tauri-build/latest/tauri_build/struct.AppManifest.html
        // and https://v2.tauri.app/security/permissions/
        tauri_build::AppManifest::new().commands(&[
```
But you can word this better than I can...

Reply:
You're right. I thought of adding this, but forgot.